src/compilers/if.cr

```diff
@@ -50,7 +68,33 @@ module Mint
           "null"
         end
 
-      "(#{condition} ? #{truthy} : #{falsy})"
+      case condition
+      when Tuple(String, Tuple(String, Array(String)), Bool)
+        tru =
+          js.iif do
+            js.statements(condition[1][1] + [
+              js.return truthy,
+            ])
+          end
+
+        if condition[2]
+          js.asynciif do
+            js.statements([
+              condition[0],
+              js.return "(#{condition[1][0]} ? #{tru} : #{falsy})",
+            ])
+          end
+        else
```

Comment:
```suggestion
        truthy =
          js.iif do
            js.statements(condition[1][1] + [
              js.return truthy,
            ])
          end

        if condition[2]
          js.asynciif do
            js.statements([
              condition[0],
              js.return "(#{condition[1][0]} ? #{truthy} : #{falsy})",
            ])
          end
        else
```

Reply:
There is already a `truthy` variable earlier in the file, don't know yet what to call this :shrug: